applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -267,7 +273,7 @@ config ENCODER_STACK_SIZE
 config AUDIO_DATAPATH_STACK_SIZE
 	int "Stack size for audio datapath thread"
 	default 4096 if AUDIO_BIT_DEPTH_16
-	default 7168 if AUDIO_BIT_DEPTH_32
+	default 8192 if AUDIO_BIT_DEPTH_32
```

Comment:
Why is this increased only for 32 bit? Did this fail a test or something, or is it remnants from debugging?

Reply:
@gWacey